### DISCUSSION TITLE
How to show content based on my database data ?

### DISCUSSION THREAD

**Discussion by sergeynilov** [Z3d0X/filament-fabricator#126] [Q&A]
I wonder if there isa way to make some hooks when blocks of data are rendered on frontend , where I could insert piecies of html code, which I render based on data from db

**Comment by Z3d0X** [comment]
Hi you could use the "mutateData" method to change the data available to the block view

https://github.com/Z3d0X/filament-fabricator/blob/2.x/docs/README.md#mutate-data
